docs/esterni/manuale_installazione/manuale_installazione.tex

```diff
@@ -0,0 +1,285 @@
+\documentclass[a4paper, 12pt]{article}
+
+\newcommand{\templates}{../../template}
+\input{\templates/style}
+\input{\templates/front_page}
+\input{\templates/history}
+\usepackage{hyperref}
+\usepackage{graphicx}
+\usepackage{placeins}
+\usepackage{listings}
+\settitolo{Manuale Installazione Shop Chain}
+\setredattori{WinningSoftwareSolution}
+\setdestuso{esterno}
+\setdescrizione{
+Manuale installazione.
+}
+
+\addversione{0.0.0}{04/04/2022}{Federico Marchi}{Redazione}{Stesura Manuale}
+
+\begin{document}
+\makefrontpage
+
+\makeversioni
+
+\tableofcontents
+\newpage
+
+\section{Introduzione}
+\label{sec:intro}
+Il presente documento ha la funzione di descrivere in dettaglio la procedura da seguire per l'installazione di tutte le componenti necessarie per il corretto funzionamento di Shop Chain.
+\\È necessario clonare la repository GitHub al seguente indirizzo: \url{https://github.com/iota97/WinningSoftwareSolution}.
+\section{Server}
+\subsection{Requisiti}
+\subsubsection{Node Js}
+Installare \href{https://nodejs.org/en/download/}{Node Js} se non è già stato installato precedentemente. Una volta effettuata correttamente l'installazione, è necessario eseguire il seguente comando direttamente nella cartella "Server" per l'installazione delle dipendenze:
+\begin{lstlisting}[language=bash]
+  $ npm install
+\end{lstlisting}
+\subsubsection{Provider}
+\label{sec:provider}
+Per ottenere il provider per la testnet di Polygon Mumbai è necessario registrarsi su \href{https://www.moralis.io}{moralis.io}. Una volta effettuata la registrazione:
```

Comment:
Aggiungerei anche il fatto che la procedura è identica per l'utilizzo in mainnet, dopotutto il manuale di installazione dovrebbe essere "realistico", mumbai è la rete di test che usiamo noi appunto per i test non per il deployment. Tuttavia per usare la mainnet serve pagare e noi non lo faremo, quindi includerei entrambe le cose spiegando come funziona per la mainnet e poi aggiungendo qualcosa come "è possibile anche usare la testnet per testare il corretto funzionamento senza pagare"